docs/release-notes/8.9.asciidoc

```diff
@@ -1,6 +1,20 @@
 [[release-notes-header-8.9.0]]
 == 8.9
 
+[discrete]
+[[release-notes-8.9.2]]
+=== 8.9.2
+
+[discrete]
+[[bug-fixes-8.9.2]]
+==== Bug fixes
+
+* Fixes a bug that stopped the additional inline actions from completing ({pull}165099[#165099]).
```

Comment:
Correct me if I'm wrong re: the location. 
```suggestion
* Fixes a bug that prevented inline actions on the Alerts page from completing ({pull}165099[#165099]).
```

Reply:
The example in the PR only showed the Alerts page, so I'm going to assume your change is correct. @semd would you mind verifying?

Reply:
Yes, the legacy actions that were fixed are only rendered with the "row-renderer" component, which is used to render the `reason` field. It is displayed in the flyout, and in the alerts table as well, so yes, both are on the Alerts page.

Reply:
Thanks for confirming! I'll merge this change then.